lsst/ts/laser/csc.py

```diff
@@ -132,7 +165,7 @@ def assert_propagating(self, action):
         self.assert_enabled("stopPropagateLaser")
         self.assert_propagating("stopPropagateLaser")
         self.model.stop()
-        self.detailed_state = LaserDetailedStateEnum.ENABLEDSTATE
+        self.detailed_state = LaserDetailedState.ENABLEDSTATE
 
     async def do_abort(self,id_data):
         """Actually does nothing and is not implemented yet.
```

Comment:
If it is not implemented you may want to raise an exception so the command is rejected. Also, this can probably be a regular method instead of a coroutine for now.